graphql_compiler/__init__.py

```diff
@@ -58,31 +58,13 @@ def graphql_to_match(schema, graphql_query, parameters, type_equivalence_hints=N
         query=insert_arguments_into_query(compilation_result, parameters))
 
 
-def graphql_to_sql(schema, graphql_query, parameters, compiler_metadata,
-                   type_equivalence_hints=None):
+def graphql_to_sql(sql_schema_info, graphql_query, parameters):
     """Compile the GraphQL input using the schema into a SQL query and associated metadata.
 
     Args:
-        schema: GraphQL schema object describing the schema of the graph to be queried
-        graphql_query: the GraphQL query to compile to SQL, as a string
+        sql_schema_info: SQLAlchemySchemaInfo used to compile the query.
+        graphql_string: the GraphQL query to compile to SQL, as a string
         parameters: dict, mapping argument name to its value, for every parameter the query expects.
-        compiler_metadata: SqlMetadata object, provides SQLAlchemy specific backend
-                           information
-        type_equivalence_hints: optional dict of GraphQL interface or type -> GraphQL union.
-                                Used as a workaround for GraphQL's lack of support for
-                                inheritance across "types" (i.e. non-interfaces), as well as a
-                                workaround for Gremlin's total lack of inheritance-awareness.
-                                The key-value pairs in the dict specify that the "key" type
-                                is equivalent to the "value" type, i.e. that the GraphQL type or
-                                interface in the key is the most-derived common supertype
-                                of every GraphQL type in the "value" GraphQL union.
-                                Recursive expansion of type equivalence hints is not performed,
-                                and only type-level correctness of this argument is enforced.
-                                See README.md for more details on everything this parameter does.
-                                *****
-                                Be very careful with this option, as bad input here will
-                                lead to incorrect output queries being generated.
-                                *****
 
     Returns:
         a CompilationResult object, containing:
```

Comment:
Do you mind fixing the docstring for CompilationResult? Specifically for SQL, compilation_result.query is not a string, but it is some SQLAlchemy object.